DSharpPlus/Net/Rest/DiscordApiClient.cs

```diff
@@ -2012,15 +2023,25 @@ internal Task DeleteRoleAsync(ulong guild_id, ulong role_id, string reason)
             return this.DoRequestAsync(this.Discord, bucket, url, RestRequestMethod.DELETE, route, headers);
         }
 
-        internal async Task<DiscordRole> CreateGuildRoleAsync(ulong guild_id, string name, Permissions? permissions, int? color, bool? hoist, bool? mentionable, string reason)
+        internal async Task<DiscordRole> CreateGuildRoleAsync(ulong guild_id, string name, Permissions? permissions, int? color, bool? hoist, bool? mentionable, string reason, Stream icon, string emoji)
         {
+            string image = null;
```

Comment:
Same here for image/emoji combo and default emotes. See previous comment on Modify method.